### PR TITLE
added google home alarm documentation

### PR DESCRIPTION
**Pull request by eliseomartelli**
**Description:**
Added the Google Home Alarms sensor documentation.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#20645
## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/sensor.google_home_alarm.markdown [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+layout: page
+title: "Google Home Alarm"
+description: "Instructions on how to integrate a BH1750 light sensor into Home Assistant."
```
Comment (dshokouhi) [inline source/_components/sensor.google_home_alarm.markdown]: May want to update the description as this is not a light sensor :)

**Comment by dshokouhi** [comment]
Asides from the minor issue, this looks good to me.

**Review by dshokouhi — CHANGES_REQUESTED** [review]
The description should be updated for the integration.

File: source/_components/sensor.google_home_alarm.markdown (hunk 1 above)
Comment (eliseomartelli) [reply source/_components/sensor.google_home_alarm.markdown]: took care of that 😎

Comment (eliseomartelli) [reply source/_components/sensor.google_home_alarm.markdown]: Netlify preview ready :)

File: source/_components/sensor.google_home_alarm.markdown [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+The `google_home_alarm` sensor platform allows you to show the next alarm and timer from a Google Home on the network.
+It uses the [GHLocalApi](https://github.com/rithvikvibhu/GHLocalApi) to retrieve the data.
+
+To use the Google Home Alarm sensor in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: google_home_alarm
+    monitored_conditions:
```
Comment (frenck) [inline source/_components/sensor.google_home_alarm.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

File: source/_components/sensor.google_home_alarm.markdown [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+{% configuration %}
+host:
+  description: The IP address of the Google Home.
+  required: true
+  type: string
+monitored_conditions:
+  description: The conditions to monitor.
+  required: false
+  type: map
```
Comment (frenck) [inline source/_components/sensor.google_home_alarm.markdown]:
```suggestion
  type: list
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @eliseomartelli!

I've reviewed it, and left some comments. Could you please take a look? 👍

**Comment by eliseomartelli** [comment]
@frenck I've fixed the configuration to reflect the code review from the component's pull request.

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by frenck** [comment]
Not stale, parent PR is still open.

**Comment by frenck** [comment]
Re-open, parent PR is still there.

**Comment by eliseomartelli** [comment]
Waiting https://github.com/home-assistant/home-assistant.io/pull/8346 to update accordingly.

**Pull request closed without merging**